### PR TITLE
Search bar HW

### PR DESCRIPTION
**Pull request by Ostafii4uk**
- [DEMO LINK](https://Ostafii4uk.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Ostafii4uk.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  height: 70px;
+  box-sizing: border-box;
+  padding-left: 62px;
+  width: 100%;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input--large:hover {
+  box-sizing: border-box;
```
Comment (solaryasha) [inline src/style.css:47]: This rule don't change on hover

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
```
Comment (solaryasha) [inline src/style.css:51]: These rules also don't change on hover

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+.input--large:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 4px;
```
Comment (solaryasha) [inline src/style.css:58]: These rules also don't change on hover

**Review by solaryasha — CHANGES_REQUESTED** [review]
Great start, let's fix some comments

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.search--form--large {
+  position: relative;
+  margin: 20px 0;
+}
+
+.img--search--large {
+  position: absolute;
+  padding: 25px 17px 28px 26px;
```
Comment (solaryasha) [inline src/style.css:34]: Avoid using such hardcoded values, you could use just `top` and `left`

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+.search--form--small {
+  position: relative;
+}
+
+.img--search--small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  padding: 15px 9px 18px 13px;
```
Comment (solaryasha) [inline src/style.css:65]: Avoid using such hardcoded values, you could use just top and left

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+.input--large:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.input--large:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 4px;
+  font-weight: 900;
```
Comment (solaryasha) [inline src/style.css:54]: Use this rule in `input` selector, not on focus

**Review by solaryasha — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 7]
```diff
@@ -11,6 +11,37 @@
...
+      </label>
+    </form>
+
+    <form action="#" class="search--form--small" data-qa="small">
+      <label class="label label--small">
+        <img
+        class="img--search img--search--small"
+        src="images/Search.svg"
+        alt="Logo search"
+        >
```
Comment (solaryasha) [inline src/index.html:37]:
```suggestion
        <img
	          class="img--search img--search--small"
	          src="images/Search.svg"
	          alt="Logo search"
        >
```

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 4px;
```
Comment (solaryasha) [inline src/style.css:30]: This rule don't change on focus

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+  position: absolute;
+  width: 11px;
+  height: 11px;
+}
+
+.input--small {
+  height: 42px;
+  box-sizing: border-box;
+  padding-left: 33px;
+  width: 100%;
```
Comment (solaryasha) [inline src/style.css:73]: You can move this property to `input` selector

**Review by solaryasha — APPROVED** [review]

**Comment by Ostafii4uk** [comment]
Accepted

**Pull request closed without merging**